content/guides/end-to-end-testing/okta-authentication.md

```diff
@@ -7,9 +7,9 @@ e2eSpecific: true
 
 ## <Icon name="graduation-cap"></Icon> What you'll learn
```

Comment:
pretty much the same feedback as https://github.com/cypress-io/cypress-documentation/pull/4882

Reply:
once we settle on #4882 ill add the same feedback here

Reply:
made same updates from #4882 in 31b0c6c52